gen3/lib/bootstrap/templates/cdis-manifest/manifest.json

```diff
@@ -24,7 +24,8 @@
     "sower": "quay.io/cdis/sower:2020.03",
     "spark": "quay.io/cdis/gen3-spark:2020.03",
     "wts": "quay.io/cdis/workspace-token-service:2020.03",
-    "tube": "quay.io/cdis/tube:2020.03"
+    "tube": "quay.io/cdis/tube:2020.03",
+    "gearbox-middleware": "quay.io/pcdc/pcdc_dev"
```

Comment:
this is the correct line: `"gearbox-middleware": "quay.io/pcdc/gearbox-middleware:pcdc_dev",`